src/club-description.md

```diff
@@ -0,0 +1,41 @@
+---
+lang: en-CA
+title: Guelph Coding Community
+subtitle: Club description
+date: "Last updated: September 2019"
+
+documentclass: scrartcl
+numbersections: false
+geometry:
+ - margin=1in
+
+urlcolor: blue
+fontsize: 11pt
+
+header-includes:
+ - \usepackage{enumitem}
+ - \usepackage{amsfonts}
+ - \setlist[itemize,1]{label=$\bullet$}
+ - \setlist[itemize,2]{label=$\circ$}
+...
+
+The Guelph Coding Community (GCC) is a student organization focused on providing students with valuable, hands-on, extracurricular tech skills which cannot normally be acquired through coursework.
+Founded in 2013, GCC facilitates student-run workshops, tutorials, demos, and presentations for students of all backgrounds and skill-levels.
+
+GCC workshops are designed by passionate students and guest speakers to teach all manner of new skills.
+These workshops are targeted at students who share a love of coding and help to prepare for hackathons and events like the annual CS Games.
+
+Anyone can sign up to run a workshop! We believe that we learn best from our peers, and want experienced students to help expand the skillsets of Guelph Coders.
+
+Learn more at <https://socis.ca/gcc>
+
+# Current Executive
+
+ - Keefer Rourke (President)
+ - Tony Trinh (Treasurer)
+ - Nathan Laundry (Directory of Sponsorship)
+ - Zaza Hashmi (Director of Advertising and Outreach)
+ - Saqif Abrar (Director of Events)
+ - Marshall Asch (Executive at Large)
+ - Isaac Wismer (Executive at Large)
+ - Dema Abu Adas (Chair of Challenge Committee)
```

Comment:
Should contact emails be added to this list?

Reply:
I like the idea of having some form of contact info in the constitution but we don't currently have GCC dedicated emails. Would we be okay with giving our personal emails for this? I'm personally okay with it, but I'd want to make sure the other execs are as well.

Reply:
I'm going to log an issue to update this once the emails are set up and this policy goes through.

Reply:
See #2